modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -724,6 +724,7 @@ public CPDefinition copyCPDefinition(long cpDefinitionId, long groupId)
 
 			newCPInstance.setUuid(PortalUUIDUtil.generate());
 			newCPInstance.setCPInstanceId(counterLocalService.increment());
+			newCPInstance.setCPInstanceUuid(PortalUUIDUtil.generate());
```

Comment:
@brianikim this should already happen automatically

Reply:
hey @marco-leo , it does not. The uuid is still the same for both original and copy without that inclusion